docs/docusaurus.config.js

```diff
@@ -0,0 +1,143 @@
+// @ts-check
+// Note: type annotations allow type checking and IDEs autocompletion
+
+const lightCodeTheme = require("./src/themes/prism-testkube-light");
+const darkCodeTheme = require("./src/themes/prism-testkube-dark");
+
+/** @type {import('@docusaurus/types').Config} */
+const config = {
+  title: "Testkube Documentation",
+  tagline:
+    "Your somewhat opinionated and friendly Kubernetes testing framework",
+  url: "https://testkube.kubeshop.io",
+  baseUrl: "/",
```

Comment:
I think you will have to change this to `/testkube` as that's the baseUrl for the Github Pages: https://kubeshop.github.io/testkube/

I believe this would brake. Perhaps you can use your personal repo to deploy this using Github Pages and try if that would fail?